### ISSUE TITLE
[typhoon] Custom social option not available in dedicated footer module template

### ISSUE THREAD

**Issue by lic0san** [getgrav/grav-premium-issues#428]
According to documentation, it is possible to create a dedicated footer module page to enable multi-language support, instead of footer theme-level configuration. However, the module template doesn't include yet the new "custom" option for social links (added in Typhoon v2.4.5).

**Comment by rhukster** [comment]
i'll check this out..

**Comment by rhukster** [comment]
Ok, this was weird.  It seems the modular footer blueprint was accidentally removed (my fault). However when restored it still didn't work as the `modular/footer.html.twig` was missing and from what I could tell, never existed!  This was added back in 2001 so I'm a little hazy about how this was tested/validated. 

Anyway i've now restored and updated the blueprint to suppor the custom social icons in a modular footer, and also created the missing twig. I broke out the social-icons.html.twig file into it's own partial, so this can be used elsewhere if required.  All of this will be in the next 2.4.6 update which will be released shortly.
